src/drivers/St7789.cpp

```diff
@@ -119,21 +129,30 @@ void St7789::FrameRateLow() {
   WriteData(0x13);
   WriteData(0x1f);
   WriteData(0x1f);
-  nrf_delay_ms(250);
+  nrf_delay_ms(300);
 }
 
 void St7789::FrameRateNormal() {
   WriteCommand(static_cast<uint8_t>(Commands::FrameRate));
   WriteData(0x00);
   WriteData(0x0f);
   WriteData(0x0f);
-  nrf_delay_ms(250);
+  nrf_delay_ms(300);
 }
 
 void St7789::DisplayOn() {
   WriteCommand(static_cast<uint8_t>(Commands::DisplayOn));
 }
 
+void St7789::PowerControl() {
+  WriteCommand(static_cast<uint8_t>(Commands::PowerControl1));
+  WriteData(0xa4);
+  WriteData(0x30);
```

Comment:
Shouldn't this be 00?
30 is 0011 0000
So AVDD is 0 = 6.4V
AVCL is 3 = -5.0V (0 would give -4.4V)
VDS is 0 = 2.19V

Reply:
I would assume that -5.0v would use less power than -4.4v, correct me if i'm wrong

Reply:
Usually a potential difference of 0 volts means no current, so the closer to zero the lower the current should be AFAICT